packages/doc/content/components/getting-started/default-native.mdx

```diff
@@ -0,0 +1,70 @@
+### Installing
+
+In order to install our design-system just run:
+
+```bash type=highlight
+yarn add @gympass/yoga
+
+// or
+
+npm install @gympass/yoga
+```
```

Comment:
I think we could place these lines in `packages/doc/content/components/getting-started/index.mdx` since the installation step is the same for both (web and native).

What do you think?

Reply:
Hi @ggdaltoso ,

Thanks for the comment, it makes perfect sense, i made the change, what do you think? :rocket: